### DISCUSSION TITLE
Is multilanguage implemented?

### DISCUSSION THREAD

**Discussion by tdjastrzebski** [PiranhaCMS/piranha.core#1913] [Ideas]
I know this has been asked many times before but is multilanguage support implemented already?  
I mean, one website, one page structure, ability to define a localized content of each page.  
Piranha CMS appears to be an excellent project, not too complicated, I would love to use it but multilanguage support to me is a must-have functionality.

**Comment by tidyui** [comment]
There is no multi-language support for pages or posts available in Piranha CMS, so you will have to use multiple site structures for this.

Best regards

**Comment by tdjastrzebski** [comment]
Mltiple sites is exactly what I am trying to avoid.

**Reply by peppelorum** [reply]
I have started a branch for this and will see where this leads, https://github.com/peppelorum/piranha.core/tree/feature/multilingual

**Comment by tidyui** [comment]
I understand that, just clarifying that multiple sites is currently the only valid approach!

Best regards
